README.md

```diff
@@ -513,12 +512,25 @@ Haskell, let's compare Relude with some of them.
 [Protolude](https://github.com/sdiehl/protolude) is one of the most popular
 alternative preludes. It's also relatively small, but:
 
-1. Protolude supports older GHC versions (from GHC 7.6.1) while `relude` only
+1. `relude` has custom HLint rules specific to it: you can use them to remove
+   redundant imports or find hints how to use functions from `relude`. Moreover,
+   the HLint rules are generated using Dhall and there is [a blog post about
+   this technique](https://kowainik.github.io/posts/2018-09-09-dhall-to-hlint).
+   This allows to maintain HLint rules much easier because it's already not an
+   easy task.
+2. One significant difference: `head` in `protolude` returns `Maybe a` while in
+   `relude` it works with `NonEmpty`.
+3. `relude` uses type-level features to provide better error messages
+   on difference from `Prelude`, and also forbid `elem` and `notElem`
+   functions for `Set` and `HashSet` (because `elem` from `Foldable` run in
+   _O(n)_ time and you can accidentally use `elem` from `Foldable` but with
+   `relude` you can't).
+4. Protolude supports older GHC versions (from GHC 7.6.1) while `relude` only
    supports from GHC 8.0.2. So if you aim ancient GHC versions, `protolude`
```

Comment:
aim to use an alternative prelude on some older versions of GHC (e.g. GHC-7.10), then `protolude` might ...

But we need to double-check this, as I saw that they were planning to drop it :thinking:

Reply:
I've checked, seems they still support from 7.6.1 ...